src/lib/Libattr/master_queue_attr_def.xml

```diff
@@ -404,6 +404,23 @@
 	<ECL>NULL_VERIFY_VALUE_FUNC</ECL>
 	</member_verify_function>
     </attributes>
+    <attributes>
+	/* QA_ATR_Comment */
+	<member_name><both>ATTR_comment </both></member_name>			<!-- "comment" --> 
+	<member_at_decode>decode_str </member_at_decode>
+	<member_at_encode>encode_str </member_at_encode>
+	<member_at_set>set_str </member_at_set>
+	<member_at_comp>comp_str </member_at_comp>
+	<member_at_free>free_str </member_at_free>
+	<member_at_action>NULL_FUNC </member_at_action>
+	<member_at_flags><both>NO_USER_SET | ATR_DFLAG_NOSAVM </both></member_at_flags>
```

Comment:
This should be MGR_ONLY_SET. Is there a reason you don't want this attribute saved?